deploy/Chart/values.yaml

```diff
@@ -114,6 +114,7 @@ dashboard:
       memory: "300Mi"
 
 database:
+  enabled: false # Enable the Postgres database install
```

Comment:
Do we need the comment?

Reply:
@kachawla asked me to add one here: https://github.com/radius-project/radius/pull/9219#discussion_r2055084287